src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,71 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        user.setLogin("abcdef@gmail.com");
+        user.setAge(22);
+        user.setPassword("abc123");
+    }
+
+    @Test
+    void expectedBehavior_SetLoginNull_NotOk() {

```

Comment:
```suggestion
    void register_nullLogin_NotOk() {

```
There are a lot of ways to name your test methods. The main point is that they should have informative names and be consistent along with other developers in your team. For this task use such convention: `<methodUnderTest>_<state>_<expectedBehavior>`; For example, if we are testing the method register with a `null` user's age the test method name should be `register_nullAge_notOk`. `notOk` is because the test expects the register method to throw an exception.

same for other tests